components/calendar/src/types.rs

```diff
@@ -706,3 +707,58 @@ impl From<usize> for IsoWeekday {
         unsafe { core::mem::transmute(ordinal) }
     }
 }
+
+/// A moment is a RataDie with a fractional part giving the time of day.
+///
+/// NOTE: This should not cause overflow errors for most cases, but consider
+/// alternative implementations if necessary.
+#[derive(Debug, Copy, Clone, PartialEq, PartialOrd)]
+pub struct Moment(f64);
```

Comment:
issue: same comment about publicity (perhaps this should be `pub(crate)`

Reply:
I made everything pub(crate)